ui/templates/main.html

```diff
@@ -1,94 +1,100 @@
-<!DOCTYPE html><html><head><title>Φορτίο v{{.Version}}</title>
-<script src="{{.ChartJSPath}}"></script>
-<script src="{{.Version}}/static/js/fortio_chart.js"></script>
-<link rel="icon" href="../favicon.ico" />
+<!DOCTYPE html>
+<html>
+<head><title>Φορτίο v{{.Version}}</title>
+  <script src="{{.ChartJSPath}}"></script>
+  <script src="{{.Version}}/static/js/fortio_chart.js"></script>
+  <link rel="icon" href="../favicon.ico"/>
 </head>
 <body style="background: linear-gradient(to right, #d8aa20 , #c75228);">
-<a href="https://fortio.org/" target="_blank"><img src="{{.LogoPath}}" alt="." height="69" width="45" align="right" /></a>
-<img src="{{.LogoPath}}" alt="." height="92" width="60" align="right" />
-<img src="{{.LogoPath}}" alt="istio logo" height="123" width="80" align="right" />
+<a href="https://fortio.org/" target="_blank"><img src="{{.LogoPath}}" alt="." height="69" width="45"
+                                                   align="right"/></a>
```

Comment:
put that one back on 1 line please, it's not more readable with align=right... on next line (it's ok to have more than 80 cols line for html)

Reply:
Done